vnext/src/Libraries/Components/CheckBox/CheckBox.windows.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+const React = require('react');
+const StyleSheet = require('../../StyleSheet/StyleSheet');
+const invariant = require('invariant');
+const processColor = require('../../StyleSheet/processColor');
+
+const nullthrows = require('nullthrows');
+const setAndForwardRef = require('../../Utilities/setAndForwardRef');
+
+import AndroidCheckBoxNativeComponent, {
+  Commands as AndroidCheckBoxCommands,
+} from './AndroidCheckBoxNativeComponent';
```

Comment:
Does the Windows component have the Android name?

Reply:
Yes we have the android name.  -- That way we can mark in our tooling that the .windows.js should just be a copy of the .android.js version.  This reduces merge cost in the future.

We'd probably try to push a change upstream to make rename  CheckBox.android.js -> CheckBox.js, and change the native component name to be more platform agnostic but since its deprecated I'm not sure its worth the effort.